src/qt/guiutil.cpp

```diff
@@ -768,23 +768,66 @@ QString formatDurationStr(int secs)
     return strList.join(" ");
 }
 
+void sortLocaleAware(QStringList &sList)
+{
+    std::sort(sList.begin(), sList.end(), [](const QString &s1, const QString &s2){
+        return s1.localeAwareCompare(s2) < 0;
+    });
+}
+
 QString formatServicesStr(quint64 mask)
 {
     QStringList strList;
 
     for (const auto& flag : serviceFlagsToStr(mask)) {
+        if (flag == "NETWORK_LIMITED"){
+        strList.append(QString::fromStdString("LIMITED"));
+        }
+        else
+        {
         strList.append(QString::fromStdString(flag));
+        }
+        sortLocaleAware(strList);
     }
 
     if (strList.size())
-        return strList.join(" & ");
+        return strList.join(" ");
     else
         return QObject::tr("None");
 }
 
+QString shortFormatServicesStr(quint64 mask)
+{
+    QStringList strList;
+
+    for (const auto& flag : serviceFlagsToStr(mask)) {
```

Comment:
Probably better to re-implement `serviceFlagsToStr` here...

Reply:
The "Bumper" will be used to add more columns to the peertablemodel.
The point of having abbreviated services flags is to conserve space so more information can be displayed in the table view. The underlying design principle is to display as much useful info as possible "at a glance" as well as allowing those columns to be sortable. Connection time and synced blocks will be added in a follow up PR. 

![Screen Shot 2020-12-02 at 9 03 14 PM](https://user-images.githubusercontent.com/152159/100954065-dd4c8e80-34e1-11eb-9cdc-43262180bbd2.png)